arq/worker.py

```diff
@@ -153,6 +154,10 @@ class Worker:
     :param after_job_end: coroutine function to run after job has ended and results have been recorded
     :param handle_signals: default true, register signal handlers,
       set to false when running inside other async framework
+    :param job_completion_wait: time to wait before cancelling tasks after a signal.
+      Useful together with `terminationGracePeriodSeconds` in kubernetes,
```

Comment:
```suggestion
      Useful together with ``terminationGracePeriodSeconds`` in kubernetes,
```
until we move to markdown, this is still rst.